src/Ordering.Infrastructure/EntityConfigurations/OrderItemEntityTypeConfiguration.cs

```diff
@@ -15,23 +15,18 @@ public void Configure(EntityTypeBuilder<OrderItem> orderItemConfiguration)
         orderItemConfiguration.Property<int>("OrderId");
 
         orderItemConfiguration
-            .Property("_discount")
-            .HasColumnName("Discount");
+            .Property(o => o.Discount);
```

Comment:
Once you stop configuring an explicit column name, all of these are no longer necessary and can be removed.

In general, it would be good to clean up the EF model configuration to be minimal and only configure what is actually necessary (but we can do further cleanup in subsequent PRs).

Reply:
Yeah no problem but that requires to Change the Order(Item) Aggragates. Implicit configuration will not work on Init only props. In order to make it work i have  added a private setter to all init only props.

Reply:
@roji  pushed the changes

Reply:
Thanks!